src/include/sof/debug.h

```diff
@@ -50,15 +50,11 @@
 
 #ifdef DEBUG_BUILD
 
-#define DEBUG_SET_FW_READY_FLAGS	\
-{					\
-	.bits = {			\
+#define DEBUG_FW_READY_FLAGS		\
 		.build = 1,		\
 		.locks = DEBUG_LOCKS,	\
 		.locks_verbose = DEBUG_LOCKS_VERBOSE,	\
-		.gdb = DEBUG_GDB,	\
-	},				\
-}
+		.gdb = DEBUG_GDB
```

Comment:
so less debug .bits set is our intention for DEBUG_BUILD?

Reply:
The debug bits stay unchanged

Reply:
The debug bits stay unchanged

Reply:
> The debug bits stay unchanged

I meant comparing to non-DEBUG_BUILD(or called normal build), why we assign less debug .bits for DEBUG_BUILD? we assign/initialize .build, .locks, .locks_verbos and .gdb for non-DEBUG_BUILD, but only .gdb is initialized for DEBUG_BUILD, is this our intention?

I know that is not introduced by your PR, I am only curious about the difference here? @lgirdwood any comment?

Reply:
@keyonjie I think it's just the way the diff is being displayed the macros is correct.